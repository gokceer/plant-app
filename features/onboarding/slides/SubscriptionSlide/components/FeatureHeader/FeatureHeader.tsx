import React from "react";
import { StyleSheet, Text } from "react-native";

const FeatureHeader = () => {
  return (
    <>
      <Text style={styles.title}>
        <Text style={styles.appTitle}>PlantApp</Text> Premium
      </Text>
      <Text style={styles.subtitle}>Access All Features</Text>
    </>
  );
};

export default FeatureHeader;

const styles = StyleSheet.create({
  title: {
    fontSize: 26,
    marginBottom: 8,
    color: "white",
    fontFamily: "Rubik",
  },
  appTitle: {
    fontFamily: "RubikBold",
    fontSize: 30,
  },
  subtitle: {
    fontSize: 17,
    fontFamily: "RubikLight",
    color: "#FFFFFFB2",
  },
});
