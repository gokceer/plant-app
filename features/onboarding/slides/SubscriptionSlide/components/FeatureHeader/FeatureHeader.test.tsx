import { render } from "@testing-library/react-native";
import React from "react";
import FeatureHeader from "./FeatureHeader";

describe("FeatureHeader", () => {
  it("renders correctly and matches snapshot", () => {
    const { toJSON } = render(<FeatureHeader />);
    expect(toJSON()).toMatchSnapshot();
  });
});
