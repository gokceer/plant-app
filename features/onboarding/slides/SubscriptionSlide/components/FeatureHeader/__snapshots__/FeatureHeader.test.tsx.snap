// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureHeader renders correctly and matches snapshot 1`] = `
[
  <Text
    style={
      {
        "color": "white",
        "fontFamily": "Rubik",
        "fontSize": 26,
        "marginBottom": 8,
      }
    }
  >
    <Text
      style={
        {
          "fontFamily": "RubikBold",
          "fontSize": 30,
        }
      }
    >
      PlantApp
    </Text>
     Premium
  </Text>,
  <Text
    style={
      {
        "color": "#FFFFFFB2",
        "fontFamily": "RubikLight",
        "fontSize": 17,
      }
    }
  >
    Access All Features
  </Text>,
]
`;
